dev/modules/handle-click.js

```diff
@@ -0,0 +1,124 @@
+import { colorLuminance } from './utils';
+import { getModal } from './handle-swal-dom';
```

Comment:
'import' is only available in ES6 (use esnext option).<br>'getModal' is defined but never used.